sdk/storage/azure-storage-blob/azure/storage/blob/__init__.py

```diff
@@ -3,6 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)  # type: str
```

Comment:
Hi Anna @annatisch 
After changed the namesapce to azure.storage.blob.changefeed, I had to add this line. I was able to run the test locally, while tox.ini is failing, do you have any idea about this?

Reply:
This file is a production file (it ends up on a customer machine), so we must not add this line, the rest is CI problems for us.

Reply:
Thanks @lmazuel! @xiafu-msft - lets remove this line and figure out how to resolve CI :)

Reply:
Thanks @lmazuel, @annatisch !
While if this line is removed, currently I haven't figured out how to import from azure.storage.blob, because the blob/\__init\__.py under azure./storage/blob/changefeed doesn't define anything.

Reply:
We use this shared namespace pattern in other places that you can use as an example. Take a look at how they handle setup.py, directory structure, dev env set up etc
- Event Hubs extensions. This package takes a hard dependency on the azure-eventhubs sdk and shares it's namespace.
https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/eventhub/azure-eventhub-checkpointstoreblob
- Telemetry extensions. This package takes a hard dependency on azure-core and shares it's namespace:
https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/core/azure-core-tracing-opentelemetry

You can keep this line in your local environment if it helps you run tests locally etc - however please remove it from checked in code.